registry/server/apps/routes/getApps.ts

```diff
@@ -23,6 +39,24 @@ const getApps = async (req: Request, res: Response): Promise<void> => {
     if (filters.q) {
         query.where('name', 'like', `%${filters.q}%`);
     }
+    if (filters.domainId) {
```

Comment:
Could you please add a test involving these queries (if it's not exits yet).

Reply:
✅

Reply:
done ✅